app/players/elves/druid.py

```diff
@@ -0,0 +1,20 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            favourite_spell: str

```

Comment:
Your signature for all classes `__init__` seems over-indented. Decrease indentation here and in other classes as well.